tests/schema_external.py

```diff
@@ -24,9 +25,10 @@
     'user': 'djtest',
     'token': '2e05709792545ce'}
 
-dj.config['cache'] = {
-    'protocol': 'file',
-    'location': '/media/dimitri/ExtraDrive1/dj-store/cache'}
+cache_path = 'dj-cache'
```

Comment:
I prefer that we would use a [temp directory](https://docs.python.org/3.6/library/tempfile.html#tempfile.mkdtemp) for the external and cache testing purpose rather than running the risk of dumping stuff into a  potentially meaningful directory during the test .

Reply:
Why is this better?

Reply:
this will create a local directory called `dj-cache` or even put new content into the directory should it already exist. It's better to not have an unnecessary side effects during testing.

Reply:
got it

Reply:
It's quite tricky to use these tempfile objects in testing.  They clean up when the object is destroyed.  I am leaving the test as is for now but we should revise our general approach to handling temporary files used during testing.